drivers/bridge/bridge.go

```diff
@@ -147,6 +147,27 @@ func (c *networkConfiguration) Validate() error {
 	return nil
 }
 
+func (c *networkConfiguration) copyDefaultConfigs(option options.Generic, config *networkConfiguration) {
```

Comment:
True. Now it really has to go inside parseNetworkGenericOptions(...) in the second switch case.